internals/secrethub/migrate.go

```diff
@@ -257,7 +257,7 @@ func (cmd *MigratePlanCommand) addDirToPlan(client secrethub.ClientInterface, pa
 
 	tree, err := client.Dirs().GetTree(path, -1, false)
 	if err == api.ErrForbidden {
```

Comment:
Could we make this `if err == api.ErrForbidden || api.IsErrNotFound(err)`, so that we don't include any other errors than those caused by missing read permission? Then we don't require to say `if ... then ....` in the error message.

Reply:
Nice catch. Adjusted properly.